plugin-server/src/main/pluginsServer.ts

```diff
@@ -428,6 +433,27 @@ export async function startPluginsServer(
             }
         }
 
+        if (capabilities.personOverrides) {
+            const postgres = hub?.postgres ?? new PostgresRouter(serverConfig)
+            const kafkaProducer = hub?.kafkaProducer ?? (await createKafkaProducerWrapper(serverConfig))
+            const overridesWriter = new DeferredPersonOverrideWriter(postgres)
+
+            personOverridesWorker = new PeriodicTask(async () => {
+                status.debug('👥', 'Processing pending overrides...')
+                const overridesCount = await overridesWriter.processPendingOverrides(kafkaProducer)
+                ;(overridesCount > 0 ? status.info : status.debug)(
+                    '👥',
+                    `Processed ${overridesCount} pending overrides.`
+                )
+            })
+
+            personOverridesWorker.promise.catch(async () => {
+                status.error('⚠️', 'Person override writer crashed! Requesting shutdown...')
+                await closeJobs()
+                process.exit(1)
+            })
```

Comment:
I'm conflicted on how to manage error handling here.

I'd generally prefer this to crash on error to avoid dealing with the complexity of error recovery. This makes sense for our usage (where we'll be running a single standalone instance of this worker) but might not make as much sense for other cases where multiple plugin service instances might be running, since the failure to acquire the lock would take down the entire server — not bad for us because we shouldn't be doing that, but bad for other uses.

The only ideas I've come up with so far here:

1. catch and log the lock acquisition error but don't crash: safer, but means that overrides could be stalled indefinitely if there is an issue with acquiring the lock -- at some point, things will need to become visibly broken
2. leader election: please no

Reply:
Also, completely OK to blow up the pod indeed, like blobby is doing.
We'll need to alert on that, but that's easy.